plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/repository/M2RepositoryService.java

```diff
@@ -98,7 +98,7 @@ private static IPath getDownloadedSourceLocation(MavenCoordinates mavenCoordinat
   }
 
   @Activate
-  protected void activate() {
+  public void activate() {
```

Comment:
You should use the OGSi service instance of `M2RepositoryService` to access it. And keep this method protected.